Assets/Scripts/LevelTransition.cs

```diff
@@ -21,6 +21,7 @@ public class LevelTransition : MonoBehaviour
 	public static void init()
 	{
 		// maxLevel = UnityEngine.SceneManagement.SceneManager.sceneCountInBuildSettings-1;
+		WaveManager.currentLevelNumber = 1;
```

Comment:
is this necessary for something

Reply:
> is this necessary for something

When I played through the game and restarted and played through a couple more levels I got an error from SpawnManager that took in the wave and level number, pretty sure its because the level number is static and doesn't get reset so it exceeds the total index length, although I could be wrong if it actually is being reset somewhere else